static/bidder-info/e_volution.yaml

```diff
@@ -0,0 +1,13 @@
+maintainer:
+  email: "admin@e-volution.ai"
+capabilities:
```

Comment:
It looks like E-Volution is a member of IAB Europe with the id 957. Please provide that information in this file for proper enforcement of GDPR when applicable.